test2.txt

```diff
@@ -0,0 +1,8 @@
+Just some ipsm lorem stuff
```

Comment:
[On the Git mailing list](https://github.com/webstech/gggmail338feca5-b0c6-e025-6e38-efde23072337@example.com), Code Bot wrote ([reply to this](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis)):

``````````email
This email is being reviewed.

   From: "Chris. Webster" <chris@webstech.net>
   
   2022-12-25 15:59:10.93
   
   test2.txt
   
   Signed-off-by: Chris. Webster <chris@webstech.net>
   ---
    test2.txt | 8 ++++++++
    1 file changed, 8 insertions(+)
    create mode 100644 test2.txt
   
   diff --git a/test2.txt b/test2.txt
   new file mode 100644
   index 0000000..fa0e7b5
   --- /dev/null
   +++ b/test2.txt
   @@ -0,0 +1,8 @@
   +Just some ipsm lorem stuff
   +for a few lines to
   +make a change
   +big enough to
   +surpass diff
   +basic line
   +limits.
   +15:59:10.93  
   -- 
   webstech
   
   
``````````